experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -1173,9 +1179,15 @@ describe("FormAudience", () => {
         fireEvent.blur(field);
       });
 
-      expect(
-        container.querySelector(`.invalid-feedback[data-for=${fieldName}]`),
-      ).toHaveTextContent(FIELD_MESSAGES.POSITIVE_NUMBER);
+      if (fieldName === "totalEnrolledClients") {
+        expect(
+          container.querySelector(`.invalid-feedback[data-for=${fieldName}]`),
+        ).toHaveTextContent(FIELD_MESSAGES.NON_ZERO_POSITIVE_NUMBER);
+      } else {
+        expect(
+          container.querySelector(`.invalid-feedback[data-for=${fieldName}]`),
+        ).toHaveTextContent(FIELD_MESSAGES.POSITIVE_NUMBER);
+      }
```

Comment:
that's not a right approach, in the test cases we should be sure about the outcome since we are setting up test case, it should not have if/else in it, Kindly adjust the test cases based on the input given for totalEnrolledClients

Reply:
Okay